pyof/foundation/network_types.py

```diff
@@ -9,7 +9,102 @@
 from pyof.foundation.constants import VLAN_TPID
 from pyof.foundation.exceptions import PackException, UnpackException
 
-__all__ = ('Ethernet', 'GenericTLV', 'IPv4', 'VLAN', 'TLVWithSubType', 'LLDP')
+__all__ = ('ARP', 'Ethernet', 'GenericTLV', 'IPv4', 'VLAN', 'TLVWithSubType',
+           'LLDP')
+
+
+class ARP(GenericStruct):
+    """ARP packet "struct".
+
+    Contains fields for and ARP header and data.
```

Comment:
An ARP packet contains header fields (as htype, ptype, etc) and data fields (sha, spa, tha and tpa).

The typo will be fixed, sorry (: